governator-core/src/main/java/com/netflix/governator/LifecycleModule.java

```diff
@@ -63,7 +63,7 @@
     @Singleton
     @SuppressLifecycleUninitialized
     static class LifecycleProvisionListener extends AbstractLifecycleListener implements ProvisionListener {
-        private final ConcurrentMap<Class<?>, TypeLifecycleActions> cache = new ConcurrentHashMap<>();
+        private final ConcurrentMap<Class<?>, TypeLifecycleActions> cache = new ConcurrentHashMap<>(1<<12);
```

Comment:
Let's just have a nice constant for this, 

static final int INITIAL_SIZE = 4096